### ISSUE TITLE
Add support for development mode

### ISSUE THREAD

**Issue by mvysny** [vaadin/vaadin-gradle-plugin#12]
Currently the plugin always uses production mode and packages WAR in such way (even if the `isProductionMode = false` is specified in build.gradle). The production-mode WAR works very well, unfortunately the project can not be launched in Intellij: a Vaadin 14.1.2-based project fail to be run in Intellij+Tomcat: the startup fails with

```
java.lang.NullPointerException: source
	at java.util.Objects.requireNonNull(Objects.java:228)
	at java.util.Scanner.<init>(Scanner.java:578)
	at com.vaadin.flow.server.frontend.FrontendUtils.getStatsAssetsByChunkName(FrontendUtils.java:548)
	at com.vaadin.flow.server.BootstrapHandler$BootstrapPageBuilder.appendNpmBundle(BootstrapHandler.java:877)
	at com.vaadin.flow.server.BootstrapHandler$BootstrapPageBuilder.setupFrameworkLibraries(BootstrapHandler.java:859)
	at com.vaadin.flow.server.BootstrapHandler$BootstrapPageBuilder.setupDocumentHead(BootstrapHandler.java:742)
	at com.vaadin.flow.server.BootstrapHandler$BootstrapPageBuilder.getBootstrapPage(BootstrapHandler.java:517)
	at com.vaadin.flow.server.BootstrapHandler.synchronizedHandleRequest(BootstrapHandler.java:458)
	at com.vaadin.flow.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)
	at com.vaadin.flow.server.VaadinService.handleRequest(VaadinService.java:1540)
	at com.vaadin.flow.server.VaadinServlet.service(VaadinServlet.java:247)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
```

Steps to reproduce:

1. git clone https://github.com/mvysny/karibu10-helloworld-application
2. cd karibu10-helloworld-application
3. git checkout vaadin14-plugin
4. Open the project in Intellij
5. Add Tomcat/Local launch configuration
6. Run the project in Tomcat.
7. The browser opens at http://localhost:8080, showing the abovementioned stacktrace.

**Comment by mvysny** [comment]
Solved when #18 is implemented.

**Comment by mvysny** [comment]
Fixed in 401b0e02b56e42288aff1ed4adbaf2b4f2ea4d8c
